src/OverviewPage/OverviewPage.tsx

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import {
+    Bullseye,
+    Button,
+    ButtonVariant,
+    Card,
+    CardBody,
+    CardFooter,
+    CardHeader,
+    CardHeaderMain,
+    CardTitle,
+    Grid,
+    Label,
+    PageSection,
+    PageSectionVariants,
+    Stack,
+    StackItem,
+    Text,
+    TextVariants,
+    Title,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import ApacheKafkaLogo from './static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Standard-RGB.svg';
+import APIMgmtLogo from './static/images/Logo-Red_Hat-OpenShift-API_Management-A-Standard-RGB.svg';
+import DataScienceLogo from './static/images/Logo-Red_Hat-OpenShift-Data_Science-A-Standard-RGB.svg';
+import ServiceRegistryLogo from './static/images/Logo-Red_Hat-OpenShift_Service_Registry-A-Standard-RGB.png';
+import DataServicesLogo from './static/images/Logo-Red_Hat-Data_Services-A-Standard-RGB.png';
+import './OverviewPage.css'
+
+import { useTranslation } from 'react-i18next';
+
+export type OverviewPageProps = {
+    onClickKafkainstance: (event: any) => void;
+    getKafkaHref: string;
+    onClickServiceRegistry: (event: any) => void;
+    getServiceRegistryHref: string;
+
+}
+
+export const OverviewPage: React.FunctionComponent<OverviewPageProps> = ({
+    onClickKafkainstance,
+    getKafkaHref,
+    onClickServiceRegistry,
+    getServiceRegistryHref,
+}) => {
+    const { t } = useTranslation();
+
+    return (
+        <>
+            {/* Hero banner with graphic */}
+            <PageSection
+                className="app-services-ui--page-section--marketing app-services-ui--banner pf-t-dark pf-u-background-color-dark-100"
+                variant={PageSectionVariants.dark}
+            >
+                <Stack hasGutter>
+                    <Title headingLevel="h1" size="2xl">
+                        {t('overview.heroTitle')}
+                    </Title>
+                    <Text className="app-services-ui--banner__tagline">{t('overview.heroTagline')}</Text>
+                    <Text component={TextVariants.p}>{t('overview.heroDescription')}</Text>
+                    <Text component={TextVariants.p} className="pf-u-mt-md">
+                        {t('overview.heroDescription2')}
+                    </Text>
+                    <StackItem>
+                        <Button
```

Comment:
We want the link to generate a link that will work correctly with react router. This is not the way to do it.

Check the [documentation| https://www.patternfly.org/v4/components/button/#router-link]. Ignore the variant, just the component prop part.